2024/ig-security.html

```diff
@@ -240,7 +240,8 @@ <h3 id="w3c-coordination">W3C Groups</h3>
              <dt><a href="https://www.w3.org/groups/ig/privacy/">Privacy Interest Group (PING)</a></dt><dd>This Interest Group will coordinate with PING for the Self-Review Questionnaire: Security and Privacy, for Threat Models related to Privacy and Harm, and to harmonize and improve horizontal reviews.</dd>
              <dt><a href="https://www.w3.org/groups/wg/webappsec/">Web Application Security Working Group (WebAppSec)</a></dt><dd>This Interest Group will coordinate with WebAppSec for developing security features and mitigations, and for Threat Models related to the Web Platform.</dd>
              <dt><a href="https://www.w3.org/community/tmcg/">Threat Modeling Community Group (TMCG)</a></dt><dd>This Interest Group will coordinate with TMCG to work on Threat Models of different types, and creating a feedback loop on the Threat Modeling guide,</dd>
-             <dt><a href="https://www.w3.org/groups/wg/apa/">Accessible Platform Architectures (APA) Working Group</a></dt><dd>This Interest Group will coordinate with APA to harmonize and improve horizontal reviews.</dd>
+             <dt><a href="https://www.w3.org/community/swag/">Security Web Application Guidelines Community Group (SWAG)</a></dt><dd>This Interest Group will coordinate with SWAG to understand the web developer's needs.</dd>
```

Comment:
"the" implies there's only one web developer.

```suggestion
             <dt><a href="https://www.w3.org/community/swag/">Security Web Application Guidelines Community Group (SWAG)</a></dt><dd>This Interest Group will coordinate with SWAG to understand web developers' needs.</dd>
```